src/main.cpp

```diff
@@ -75,12 +91,13 @@ void setup()
     #endif
     #ifdef BLUEPILL
     SPI.setMISO(SPI_MISO);
-    SPI.setMOSI(SPI_MOSI);
-    SPI.setSCLK(SPI_SCK);
+	SPI.setMOSI(SPI_MOSI);
+	SPI.setSCLK(SPI_SCK);
```

Comment:
indentation plz

Reply:
I'll push `.editorconfig` file after we do this PR so it's easier for us

Reply:
:+1:

Reply:
I assume you men no indentation?